docs/shared/reference/environment-variables.md

```diff
@@ -2,30 +2,31 @@
+| NX_TASKS_RUNNER                   | string  | The name of task runner from the config to use. Can be overridden on the command line with `--runner`. Preferred over NX_RUNNER.                                                                                                       |
+| NX_TASKS_RUNNER_DYNAMIC_OUTPUT    | boolean | If set to `false`, will use non-dynamic terminal output strategy (what you see in CI), even when you terminal can support the dynamic version                                                                                          |
+| NX_VERBOSE_LOGGING                | boolean | If set to `true`, will print debug information useful for troubleshooting                                                                                                                                                              |
+| NX_DRY_RUN                        | boolean | If set to `true`, will perform a dry run of the generator. No files will be created and no packages will be installed.                                                                                                                 |
+| NX_INTERACTIVE                    | boolean | If set to `true`, will allow Nx to prompt you in the terminal to answer some further questions when running generators.                                                                                                                |
+| NX_GENERATE_QUIET                 | boolean | If set to `true`, will prevent Nx logging file operations during generate                                                                                                                                                              |
+| NX_PREFER_TS_NODE                 | boolean | If set to `true`, Nx will use ts-node for local execution of plugins even if `@swc-node/register` is installed.                                                                                                                        |
+| NX_IGNORE_CYCLES                  | boolean | If set to `true`, Nx will ignore errors created by a task graph circular dependency. Can be overriden on the command line with `--nxIgnoreCycles`                                                                                      |
+| NX_BATCH_MODE                     | boolean | If set to `true`, Nx will run task(s) in batches for executors which support batches.                                                                                                                                                  |
+| NX_BUILDABLE_LIBRARIES_TASK_GRAPH | boolean | If set to `true`, Nx will use the task graph to identify the dependent buildable libraries' targets instead of relying on finding dependent libraries with targets with the same name.                                                 |
```

Comment:
Why is this not the default?

Reply:
We'll first advise to try it out in a couple of reported issues to gather feedback and ensure we don't break existing setups. We'll switch the default soon.

Reply:
I'll hold this off. I'm gonna wait to see if some folks reply in the issues I pinged about using this. Based on the feedback, we might switch the default, in which case, I'll rename the variable to mean the opposite. We'll keep for a while as a escape hatch and once we're confident it works, we'll remove it.